packages/ui/Button/Button.module.scss

```
@@ -0,0 +1,57 @@
+// Button.module.scss
+//
+@use "../styles/mixins";
+
+.button {
+	display: inline-block;
+	box-sizing: border-box;
+	position: relative;
+	overflow: visible;
+	white-space: nowrap;
+	width: auto;
+	z-index: 0;
+	cursor: pointer;
+	-webkit-tap-highlight-color: transparent;
+
+	//**
+	//* Classes to apply to the background of the button, used on a child of button.
+	//* @public
+	//*/
+	.bg {
+		@include mixins.position(0);
+		position: absolute;
+		z-index: -1;
```

Isn't this supposed to be wrapped in &?
```suggestion
		& {
		    position: absolute;
		    z-index: -1;
		}
```